clients/cobol-lsp-vscode-extension/src/extension.ts

```diff
@@ -58,9 +58,14 @@ export async function activate(context: vscode.ExtensionContext) {
         () => {
         gotoCopybookSettings();
     }));
-    context.subscriptions.push(vscode.commands.registerCommand("cobol-lsp.commentLine.toggle", () => { commentCommand(CommentAction.TOGGLE) }));
-    context.subscriptions.push(vscode.commands.registerCommand("cobol-lsp.commentLine.comment", () => { commentCommand(CommentAction.COMMENT) }));
-    context.subscriptions.push(vscode.commands.registerCommand("cobol-lsp.commentLine.uncomment", () => { commentCommand(CommentAction.UNCOMMENT) }));
+    context.subscriptions.push(vscode.commands.registerCommand("cobol-lsp.commentLine.toggle",
+     () => { commentCommand(CommentAction.TOGGLE); }));
+    context.subscriptions.push(vscode.commands.registerCommand("cobol-lsp.commentLine.comment",
+    () => { commentCommand(CommentAction.COMMENT); }));
+    context.subscriptions.push(vscode.commands.registerCommand("cobol-lsp.commentLine.uncomment",
+    () => { commentCommand(CommentAction.UNCOMMENT); }));
+    context.subscriptions.push(vscode.commands.registerCommand("cobol-lsp.snippets.insertSnippets",
+    async () => { pickSnippet(); }));
```

Comment:
The async here is unnecessary.  Actually this should work just well
```suggestion
    pickSnippet
```